cmd/dev.go

```diff
@@ -399,13 +402,19 @@ func (cmd *DevCmd) startServices(f factory.Factory, config *latest.Config, gener
 	}
 
 	// Open UI if configured
-	if cmd.UI {
+	if cmd.UI || cmd.UIPort != 0 {
 		cmd.UI = false
 		logger.StartWait("Starting the ui server...")
 		defer logger.StopWait()
 
+		var port *int
+		if cmd.UIPort != 0 {
+			port = ptr.Int(cmd.UIPort)
+		}
+		cmd.UIPort = 0
```

Comment:
why is that necessary?

Reply:
It was to prevent the server from starting twice. Because of another change, it's not needed anymore.